slick-testkit/src/main/scala/com/typesafe/slick/testkit/tests/OldPlainSQLTest.scala

```diff
@@ -110,15 +110,55 @@ class OldPlainSQLTest extends TestkitTest[JdbcTestDB] {
     println("User for ID 2: "+res)
     assertEquals(User(2,"guest"), res)
 
-    val s1 = sql"select id from USERS where name = ${"szeiger"}".as[Int]
-    val s2 = sql"select id from USERS where name = '#${"guest"}'".as[Int]
-    assertEquals("select id from USERS where name = ?", s1.getStatement)
+    val s1 = sql"select id from USERS where name = '${"szeiger"}'".as[Int]
+    val s2 = sql"select id from USERS where name = '${"guest"}'".as[Int]
+    assertEquals("select id from USERS where name = 'szeiger'", s1.getStatement)
```

Comment:
The interpolation works just a little differently. If it is able to detect a constant it binds it into the statement and any variable is left as a parameter. This will safeguard against SQL injections since variables will never be bound into the statement. This is simpler since the programmer need not worry about placing a '#' for controlling the bind.